pkg/cluster/cluster.go

```diff
@@ -133,6 +133,10 @@ func new(config Config, e *spec.EtcdCluster, stopC <-chan struct{}, wg *sync.Wai
 		if err := c.createClientServiceLB(); err != nil {
 			return nil, fmt.Errorf("fail to create client service LB: %v", err)
 		}
+	} else {
+		if err := c.bm.checkSerivce(); err != nil {
```

Comment:
this is not descriptive. it is not clear to me why we need this code when reading it through.

Now it looks like

```go
if oldCluster {
    c.bm.checkService()
}
```

I am asking myself why we need to check service if this is an old cluster? what service is it checking?